config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  get 'welcome/index'
+
+  root to: 'welcome#index'
+  
+  resources :drivers
+
+  resources :passengers do 
+    resources :trips, only: [:index, :new, :show, :create]
```

Comment:
I'm not convinced that you need any of these nested routes except for `:create`, so it might be good to remove `:index, :new, :show` :) Let me know if you have questions on that.